src/main/java/com/cascadebot/cascadebot/commands/fun/CatCommand.java

```diff
@@ -9,31 +9,30 @@
 import com.cascadebot.cascadebot.commandmeta.CommandType;
 import com.cascadebot.cascadebot.commandmeta.ICommandMain;
 import com.cascadebot.cascadebot.permissions.Permission;
-import com.google.gson.JsonObject;
 import net.dv8tion.jda.core.EmbedBuilder;
 import net.dv8tion.jda.core.entities.Member;
-import okhttp3.*;
+import okhttp3.OkHttpClient;
+import okhttp3.HttpUrl;
+import okhttp3.Request;
+import okhttp3.Response;
 import org.json.JSONArray;
 import org.json.JSONObject;
 import java.io.IOException;
 
-
 public class CatCommand implements ICommandMain {
 
     @Override
     public void onCommand(Member sender, CommandContext context) {
-        OkHttpClient client = new OkHttpClient();
 
```

Comment:
No blank line here :D

Reply:
There is on my screen? and no files to commit so eh?

Reply:
I mean there shouldn't be a blank line here :P